tests/bluetooth/bsim_bt/bsim_test_resume_adv/central_hr/boards/rv32m1_vega_ri5cy.overlay

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright 2019 NXP
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+&lptmr1 {
+	interrupt-parent = <&intmux0_ch2>;
+};
+
+&intmux0_ch2 {
+	status = "okay";
+};
+
+&intmux0_ch3 {
+	status = "okay";
+};
+
+&generic_fsk {
+	interrupt-parent = <&intmux0_ch3>;
+	status = "okay";
+};
```

Comment:
Why are we adding board overlays for boards that are not the `nrf52_bsim` board in the BSIM test directory?